test/e2e/operator_test.go

```diff
@@ -121,3 +132,57 @@ func TestDefaultContour(t *testing.T) {
 		t.Fatalf("failed to delete namespace %s: %v", defaultContourNs, err)
 	}
 }
+
+func TestContourSpecNs(t *testing.T) {
+	testName := "test-user-contour"
+	specNs := fmt.Sprintf("%s-%s", defaultContourNs, testName)
+	removeNs := true
+	cntr, err := newContour(ctx, kclient, testName, operatorNs, specNs, removeNs)
+	if err != nil {
```

Comment:
might be worth pulling in the [testify](https://github.com/stretchr/testify) `assert` and `require` packages here -- helps cut down the error-checking verbosity a bit. `assert` will log errors, and `require` will use fatal (log and exit immediately). Then all the `if err != nil { ...` checks can be replaced with `require.NoError(t, err, <msg>)`.

Reply:
Yeah, we have an issue created for using testify https://github.com/projectcontour/contour-operator/issues/13.